.github/expected-missing-return-types.diff

```diff
@@ -1,15 +1,15 @@
 # Run these steps to update this file:
 sed -i 's/ *"\*\*\/Tests\/"//' composer.json
 composer u -o
-SYMFONY_PATCH_TYPE_DECLARATIONS='force=2' php .github/patch-types.php
+SYMFONY_PATCH_TYPE_DECLARATIONS='force=2&php=8.0' php .github/patch-types.php
 head=$(sed '/^diff /Q' .github/expected-missing-return-types.diff)
 (echo "$head" && echo && git diff -U2 composer.json src/) > .github/expected-missing-return-types.diff
```

Comment:
the diff on this file is mine, I ran the listed step again from scratch to keep the file in sync